service/src/main/java/com/epam/esm/service/exception/MostWidelyTagNotExistsException.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.esm.service.exception;
+
+public class MostWidelyTagNotExistsException extends ServiceException {
```

Comment:
I don't understand why you need this exception. If there is no such tag, you are returning empty list of tags or empty object

Reply:
Thank you. I've corrected this place in my code. (fixed)